contracts/ERC721BridgelessMinting.sol

```diff
@@ -12,7 +12,6 @@ import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
  *  cannot own any slot
  * @dev the null address is used only as the implicit target address when
  *  executing the burn method"
- * @dev The contract has the possibility of adding default operator that can
```

Comment:
these lines are now a mess... with a line extra should be removed, there is a " at the end of: with method", there is a typo chang -> changed...